### DISCUSSION TITLE
Git push origin master

### DISCUSSION THREAD

**Discussion by lucasdurannn** [community/community#84054] [Actions]
### Select Topic Area

Bug

### Body

I'm trying to use the git push origin master command, but it's giving me this message:
![Screenshot_44](https://github.com/community/community/assets/126881615/4a6bd8bb-1051-48c2-8e4c-f24b55152604)

Username for 'https://github.com': lucasdurannn
Password for 'https://lucasdurannn@github.com':
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/lucasdurannn/transparencia.git/'

but my friends are able to use the command normally.

**Comment by elbrant** [comment]
This is how I solved the problem:

1. [switch from HTTPS to SSH](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#switching-remote-urls-from-https-to-ssh)
2. [change your user.email config to your "github noreply" email address (if you wanted to keep your email private)](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories#switching-remote-urls-from-https-to-ssh)
3. navigate into your email settings on github: login > click on your avatar in the upper right corner > settings > email
4. uncheck the "Block command line pushes that expose my email" checkbox

And you should be good to go!!
